### PR TITLE
XSound will play locally while holding down cmd or alt key

### PR DESCRIPTION
**Pull request by mswiechowicz**

### CODE REVIEW

File: packages/client/src/sound-board/service/soundboard-service.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const audioClient = new Audio();
```
Comment (mswiechowicz) [inline packages/client/src/sound-board/service/soundboard-service.ts:1]: It's here because it would be nice to have just one instance of the new Audio

**Comment by mswiechowicz** [comment]
resolves #198

File: packages/client/src/sound-board/service/soundboard-service.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
+const AudioClient = new Audio();
```
Comment (mswiechowicz) [inline packages/client/src/sound-board/service/soundboard-service.ts:1]: It's here because it would be nice to have just one instance of the new Audio

Comment (Deseteral) [reply packages/client/src/sound-board/service/soundboard-service.ts:1]: This is fine :) But I think it should be lower case

```suggestion
const audioClient = new Audio();
```

File: packages/client/src/sound-board/service/soundboard-service.ts [hunk 3]
```diff
@@ -9,6 +10,9 @@ async function queueXSound(name: string) {
   });
 }
 
-export {
-  queueXSound,
-};
+function playXSoundLocally(url: string) {
+  AudioClient.src = url;
+  AudioClient.play();
```
Comment (Deseteral) [inline packages/client/src/sound-board/service/soundboard-service.ts:15]: Shouldn't this technically be called inside a `load` event?
I think that there is a possibility for `.play()` to be called before the sound is downloaded.

**Review by Deseteral — APPROVED** [review]

**Pull request merged**